src/Page.js

```diff
@@ -587,13 +587,14 @@ Page.prototype.insertSiteNav = function (pageData) {
     const siteNavHtml = md.render(siteNavDataSelector('navigation').html().trim().replace(/\n\s*\n/g, '\n'));
     // Add Bootstrap padding class to rendered unordered list
     const siteNavHtmlSelector = cheerio.load(siteNavHtml, { xmlMode: false });
-    siteNavHtmlSelector('ul').first().addClass('px-3');
+    siteNavHtmlSelector('ul').first().addClass('px-0');
+    siteNavHtmlSelector('ul ul').addClass('pl-3');
```

Comment:
Good enhancement to get rid of `margin-left: -1em` from the `site-nav-list` class.

So far I tested with deeply nested and no nesting `site-nav`s and so far works well.

Maybe we should change it to `pl-1` to save some space for text?

Reply:
Thanks for bringing it up!

Personally, I feel that the indentation for `pl-1` and `pl-2` isn't distinct enough. Here are some images of the different indentation levels for comparison:

`pl-1`:
<img width="279" alt="Screenshot 2019-04-04 at 2 04 37 PM" src="https://user-images.githubusercontent.com/17447681/55537431-c3e0b700-56ee-11e9-92d1-ae2f7a56a063.png">

`pl-2`:
<img width="279" alt="Screenshot 2019-04-04 at 2 04 46 PM" src="https://user-images.githubusercontent.com/17447681/55537437-c93e0180-56ee-11e9-840a-0a7b01d0ee3e.png">

`pl-3`:
<img width="276" alt="Screenshot 2019-04-04 at 2 04 54 PM" src="https://user-images.githubusercontent.com/17447681/55537446-cfcc7900-56ee-11e9-83a7-2fd073245054.png">

Reply:
If there are no more changes, you can remove [WIP] from the PR title.

Reply:
Done!